src/encoder.rs

```diff
@@ -1780,7 +1780,13 @@ fn encode_partition_topdown<T: Pixel, W: Writer>(
       if split_horz { partition_types.push(PartitionType::PARTITION_HORZ); };
       if split_vert { partition_types.push(PartitionType::PARTITION_VERT); };
     } else if bsize.width_log2() == fi.min_partition_size.width_log2() + 1 {
-      partition_types.extend_from_slice(RAV1E_PARTITION_TYPES);
+      partition_types.push(PartitionType::PARTITION_NONE);
+      partition_types.push(PartitionType::PARTITION_SPLIT);
```

Comment:
I'd like to share that changing the order of partition type causes a change in bitstream thus in AWCY bd-rate results (ignorable, < 0.02%, presumably due to equality condition). https://beta.arewecompressedyet.com/?job=master-c84872d70f7fd741d7c3c03cf44cf5e4cf1ea21d&job=master-bbb8e24938cf5fc047ce43f4fd55b95e62e0c953

Please note that changing the order of partition type test is fine with current topdown strategy but not allowed with bottomup fashion.

Reply:
Thank you for pointing this out, will fix.

Reply:
It seems in the `must_split` case if we use the same order as `RAV1E_PARTITION_TYPES` there is actually a small loss in average BD-rate (0.14).